### DISCUSSION TITLE
Can you please make available your Obsidian "structure", e.g. indexing, tagging, TOC's, etc

### DISCUSSION THREAD

**Discussion by PaulFWatts** [BryanJenksCommunity/FAQ#8] [Q&A] [answered]
### Question

Can you please make available your Obsidian "structure", e.g. indexing, tagging, TOC's, etc

---

By filing a question to this repo, I promise that

-   [x] I have searched open/closed issues on this repo for my answer already

I understand that my issue may be closed if I don't fulfill my promises.

**Comment by tallguyjenks** [comment, marked as answer]
Index file is [[HERE]](https://github.com/BryanJenksCommunity/FAQ/blob/master/Files/INDEX.md)
My templates are here: #4 :)

**Reply by mnott** [reply]
Thanks Brian. What in addition would we need to make it work? It looks kind of garbled when just dropping into a vault.

**Reply by tallguyjenks** [reply]
I'm unsure what you mean? What's garbled? What is not working that need to work?

**Reply by mnott** [reply]
Thanks Bryan, it was my lack of understanding. I've made it work. Now, it would be great if you could come up with an actual explanation :) What are all those emojis really for, and how are you actually using those templates?

**Reply by tallguyjenks** [reply]
My emoji's i explain in my [[[taxonomy note]]](https://publish.obsidian.md/bryan-jenks/Tag+Taxonomy) elaborate on in my [video about tags v.s. links](https://youtu.be/zIh1S7ra3aI) and another for using [templates](https://youtu.be/1eUxQo6Dy7k) hopefully this answer your questions

**Comment by PaulFWatts** [comment]
All very helpful and appreciated. Thanks Bryan!
